docusaurus/docs/how-to-guides/ui-extensions/create-an-extension-point.md

```diff
@@ -17,7 +17,7 @@ keywords:
 
 import ExtensionPoints from '@shared/extension-points.md';
 
-An "extension point" is a part of your plugin UI or Grafana UI where other plugins can hook in with either links or React components, to extend the user experience based on a context exposed by the extension point.
+An _extension point_ is a part of your plugin UI Grafana UI where other plugins can hook in links or React components. You can use them to extend the user experience based on a context exposed by the extension point.
```

Comment:
we actually need the `or` as it can be from a plugin or from Grafana

Reply:
yes, good suggestions, done